src/core/Akka.TestKit/TestKitBase_ExpectAllWithPredicates.cs

```diff
@@ -21,7 +21,7 @@ public IReadOnlyCollection<object> ExpectMsgAllOfMatchingPredicates(params Predi
     {
         return ExpectMsgAllOfMatchingPredicatesAsync(predicates, cancellationToken)
             .ToListAsync(cancellationToken)
-            .ConfigureAwait(false).GetAwaiter().GetResult();
+            .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.